.expeditor/scripts/shared.sh

```diff
@@ -68,7 +85,6 @@ install_latest_hab_binary() {
     else
```

Comment:
Somehow I think the original intent of `hab_binary` got lost... it was supposed to be the concrete path to a `/hab/pkgs/core/hab/XXX/XXXXXXXXXXXX/bin/hab` binary, so we could ensure we used the one we just installed, rather than whatever was curlbashed into the `PATH`.

After all, if `hab_binary` is always pointing to the on-PATH location that it gets curlbashed to, there isn't much value in setting it in a variable... you'd just always call `hab` directly.

As such, if we're going to be setting `hab_binary` it should be in this function, and it should be the complete path to that `hab` binary.

I think we should probably also rename this function to better reflect what it's doing. Thinking a bit more, we should probably completely remove the call to `curlbash_hab` from inside this function. That will further help to clarify the distinct responsibilities the two functions have.